canonical-websites/microk8s.io
# Add Dockerfile for production image

**Based on #1 - merge that first**

I'm adding the dockerfile for building the production image.

Closes #4

QA
==

``` bash
docker build --tag microk8s --build-arg COMMIT_ID=`git rev-parse HEAD` .
docker run -ti -p 8099:80 microk8s
```

Now go to http://127.0.0.1:8099 and see the site. Go to http://127.0.0.1:8099/_status/check and see "OK".

Also try `curl -I http://127.0.0.1:8099` and see the X-Commit-ID header.
